doc/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Ongoing development
 
 For F3D users:
+ - Added new options `--hdri-file`, `hdri-ambient`, `hdri-skybox` to have more control on the HDRI behavior
```

Comment:
no need of these extra `--` just `hdri-file` should be good for the changelog, right?

(or add it for the ambient `--hdri-ambient` and skybox `--hdri-skybox`)

Reply:
good point. wdyt @Meakk ? Keep the dashes ?

Reply:
I'd omit the dashes

Reply:
please note this is not how 2.1.0 release note was written.